src/components/HourlyWeather/HourlyWeather.tsx

```diff
@@ -1,20 +1,20 @@
 import { useContext } from "react";
-import { hours } from "../../api/api";
 import { WeatherContext } from "../../store/weather";
 import Hours from "../Hours/Hours";
 import { HourlyContainer } from "./style";
-
+import { Hour } from "../../types/types";
```

Comment:
you can do like import type { Hour } from "../../types/types"; to highlight that `Hour` is a type not a component